docs/roadmap.md

```diff
@@ -96,35 +96,34 @@ For questions or status updates on either features listed here, or features not
-| Support multiple top-level windows on separate threads | &#128311;                              | &#128311;                             | &#128311;                           | &#128994;                         |
-| Drag and drop                                          | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| RenderTargetBitmap                                     | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| Mouse cursor customization                             | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| Animated Gif support                                   | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| VirtualSurfaceImageSource (VSIS) support               | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| In-app acrylic                                         | &#128994;                              | &#128994;                             | &#128994;                           | &#128994;                         |
-| Background acrylic                                     |             |                          |                 |&#128994;                         |                       |
-| Mica                                     |                                 |                                |                   | &#128994;                         |                       |
-| XAML Islands                                           |                                 |                                |                    | &#128994;                         |
-| Media Controls (e.g. `<MediaPlayerElement>`)             |                                 |                                |                              | &#128994;                         |
-| `<InkCanvas>`                                            |                                 |                                |                              | &#128994;                         |
-| `<MapControl>`                                           |                                 |                                |                              |  &#129002;                     |
+**WinUI 3 - Windows App SDK 1.0** is our latest release. 
+
+| Customer Capability                                    | Project Reunion 0.5 <br/>(March 2021) | Windows App SDK 0.8 <br/>(June 2021) | Windows App SDK 1.0 <br/> | Windows App SDK 1.1 (still under discussion) <br/> | Planned for a future update |
```

Comment:
We should put (Nov 2021) under the 1.0 column header, which would be consistent with how 0.5 and 0.8 are listed.

For 1.1, I'd put "Planned for 1.1" as the full header.  (I would not put a parenthetical timeframe under it -- they can get that from the reunion GitHub)

Reply:
Sorry, by "Nov 2021", I meant spell out November 2021 fully in parentheses.